docs/guides/using_ocio/compatible_software.rst

```diff
@@ -387,6 +387,16 @@ Documentation :
 - `VRayTexOCIO <https://docs.chaosgroup.com/display/VRAY4MAYA/VRayTexOCIO>`__
 
 
+cineSync (ftrack)
```

Comment:
The list follows the alphabetic order. Could move the paragraph?

Reply:
This should be fixed now